docs/semgrep-supply-chain/upgrade-guidance.md

```diff
@@ -0,0 +1,125 @@
+---
+slug: upgrade-guidance
+title: Upgrade guidance
+hide_title: true
+description: Know if a vulnerable package or dependency can be easily and reliably upgraded to a fixed version.
+tags:
+  - Semgrep Supply Chain
+  - Semgrep AppSec Platform
+---
+
+import ViewDetailsSsc from "/src/components/procedure/_view-details-ssc.md"
+ 
+# Upgrade guidance
+
+Know if you or your developers can safely and reliably update a vulnerable package or dependency to a fixed version. From there, choose to:
+
+- Have Semgrep open a pull request (PR)
+- Create a Jira ticket
+- Set the finding's triage status as **To fix**
+
+Semgrep's dependency upgrade guidance uses AI to analyze if a finding can be **safely upgraded** or if upgrading the package can cause **breaking changes**. Semgrep can also create a PR to upgrade the package.
```

Comment:
```suggestion
Semgrep's dependency upgrade guidance uses [Assistant](/semgrep-assistant/overview) to analyze if a finding can be **safely upgraded** or if upgrading the package can cause **breaking changes**. Semgrep can also create a PR to upgrade the package.
```

Feel free to disregard if you think appropriate, but I thought it would be a good time to cross-link to Assistant